src/CommunityToolkit.Maui/Behaviors/AnimationBehavior.shared.cs

```diff
@@ -63,31 +69,27 @@ public BaseAnimation? AnimationType
 		get => (BaseAnimation?)GetValue(AnimationTypeProperty);
 		set => SetValue(AnimationTypeProperty, value);
 	}
+	/// <summary>
+	/// Whether a TapGestureRecognizer is added to the control or not
+	/// </summary>
+	public bool AnimateOnTap
+	{
+		get => (bool)GetValue(AnimateOnTapProperty);
+		set => SetValue(AnimateOnTapProperty, value);
+	}
 
 	/// <inheritdoc/>
 	protected override void OnAttachedTo(VisualElement bindable)
 	{
 		base.OnAttachedTo(bindable);
 
-		if (!string.IsNullOrWhiteSpace(EventName))
+		if (AnimateOnTap && bindable is IGestureRecognizers gestureRecognizers)
 		{
-			return;
-		}
+			tapGestureRecognizer = new TapGestureRecognizer();
+			tapGestureRecognizer.Tapped += OnTriggerHandled;
 
-		if (bindable is ITextInput)
-		{
-			throw new InvalidOperationException($"Animation Behavior can not be attached to {nameof(ITextInput)} without using the EventName property.");
+			gestureRecognizers.GestureRecognizers.Add(tapGestureRecognizer);
 		}
```

Comment:
Wasnt sure why this check was needed

Reply:
Have you confirmed that `AnimationBehavior` can now be attached to an `ITextInput` without using the `EventName` property? 

I'm cool removing this as long as we have confirmation on each platform (iOS, Android, Windows, MacCatalyst and Tizen).

Reply:
@TheCodeTraveler tested windows, android and iOS. All seem to work fine.

I don't work with MacCatalyst or Tizen, so I have updated the sample app, hoping someone else can test those platforms.

Only thing added to the page is the following _(VS is auto formatting the rest)_
```xaml
<Entry Margin="16,0" Placeholder="Animate on tap">
    <Entry.Behaviors>
        <mct:AnimationBehavior AnimateOnTap="True">
            <mct:AnimationBehavior.AnimationType>
                <behaviorPages:SampleScaleAnimation Easing="{x:Static Easing.Linear}" Length="100" />
            </mct:AnimationBehavior.AnimationType>
        </mct:AnimationBehavior>
    </Entry.Behaviors>
</Entry>
```

https://github.com/user-attachments/assets/46081c82-a024-4a0b-8a52-b5ebf2dc2c74